postgres-appliance/bootstrap/clone_with_basebackup.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import argparse
+from collections import namedtuple
+import logging
+import os
+import subprocess
+
+logging.basicConfig(format='%(asctime)s %(levelname)s: %(message)s', level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def read_configuration():
+    parser = argparse.ArgumentParser(description="Script to clone from another cluster using pg_basebackup")
+    parser.add_argument('--scope', required=True, help='target cluster name')
+    parser.add_argument('--datadir', required=True, help='target cluster postgres data directory')
+    parser.add_argument('--bindir', required=True, help='the directory with pg_basebackup')
```

Comment:
Originally, those scripts were not supposed to be tied to Spilo, therefore, no guarantees PATH will be set (as it is set by the Spilo Docker image). While they are now part of Spilo, I'd prefer to make as few assumptions as possible on the environment they run at (they still assume wal-e is accessible in PATH, but that is a reasonable assumption for now).